sample/net/grpc/connector/replica-connector.go

```diff
@@ -97,14 +98,23 @@ type messageStreamHandler struct {
 // HandleMessageStream implements actual communication with remote
 // replica by means of the gRPC connection established to the replica.
 func (h *messageStreamHandler) HandleMessageStream(in <-chan []byte) (<-chan []byte, error) {
+	var stream proto.Channel_ChatClient
+	var wg sync.WaitGroup
 	out := make(chan []byte)
 
-	stream, err := h.client.Chat(context.Background())
-	if err != nil {
-		return nil, err
-	}
+	wg.Add(1)
+	go func() {
+		var err error
+		stream, err = h.client.Chat(context.Background(), grpc.FailFast(false))
```

Comment:
We might want to update `grpc` package to v1.18.0 and use `grpc.WaitForReady` in favour of deprecated `grpc.FailFast`. You can use `govendor fetch google.golang.org/grpc/...@v1.18.0` from `sample` directory to do the update.